mmtrack/models/backbones/mixformer_backbone.py

```diff
@@ -0,0 +1,733 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import collections.abc as container_abcs
+import logging
+from collections import OrderedDict
+from functools import partial
+from itertools import repeat
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from einops import rearrange
+from einops.layers.torch import Rearrange
+from mmcv.runner import BaseModule
+from mmdet.models.builder import BACKBONES
+from timm.models.layers import DropPath, trunc_normal_
+
+from .utils import FrozenBatchNorm2d
+
+
+# From PyTorch internals
+def _ntuple(n):
+
+    def parse(x):
+        if isinstance(x, container_abcs.Iterable):
+            return x
+        return tuple(repeat(x, n))
+
+    return parse
+
+
+to_1tuple = _ntuple(1)
+to_2tuple = _ntuple(2)
+to_3tuple = _ntuple(3)
+to_4tuple = _ntuple(4)
+to_ntuple = _ntuple
+
+
+class LayerNorm(nn.LayerNorm):
+    """Subclass torch's LayerNorm to handle fp16."""
+
+    def forward(self, x: torch.Tensor):
+        orig_type = x.dtype
+        ret = super().forward(x.type(torch.float32))
+        return ret.type(orig_type)
+
+
+class QuickGELU(nn.Module):
+
+    def forward(self, x: torch.Tensor):
+        return x * torch.sigmoid(1.702 * x)
+
+
+class Mlp(nn.Module):
```

Comment:
Please use the existing FFN module in MMCV: https://github.com/open-mmlab/mmcv/blob/45fa3e44a22cd0b74071237304bdfef97b5b9bd6/mmcv/cnn/bricks/transformer.py#L555